backend/config.py

```
@@ -123,18 +118,22 @@ def from_env(cls) -> "Config":
             if environment.lower() == "production":
                 errors.append("SECRET_KEY is required in production environment")
             else:
-                secret_key = "09d25e094faa6ca2556c818166b7a9563b93f7099f6f0f4caa6cf63b88e8d3e7" # Fallback for dev only
+                secret_key = "09d25e094faa6ca2556c818166b7a9563b93f7099f6f0f4caa6cf63b88e8d3e7"  # Fallback for dev only
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "secret_key"

(S105)